content/syllabuses/react-specialisation.md

```diff
@@ -5,13 +5,13 @@ _db_id: 30
 
 - {{< contentlink path="topics/web-frontend/react/intro-to-react" >}}
 - {{< contentlink path="topics/watch-this-space" >}}
-- {{< contentlink path="projects/tilde-mockups" flavour="javascript,react" >}}
+- {{< contentlink path="projects/tilde-mockups" flavour="react,javascript" >}}
 - {{< contentlink path="topics/web-frontend/react/redux-architecture" >}}
 - {{< contentlink path="topics/react-redux-dev-tools" >}}
 - {{< contentlink path="topics/web-frontend/react/unit-testing" >}}
-- {{< contentlink path="projects/redux-intro/part-3" flavour="javascript" >}}
-- {{< contentlink path="projects/memory-game/part-2" flavour="react,redux,javascript" >}}
-- {{< contentlink path="projects/recipe-search/part-2" flavour="javascript" >}}
+- {{< contentlink path="projects/redux-intro/part-3" flavour="react,javascript" >}}
+- {{< contentlink path="projects/memory-game/part-2" flavour="react,javascript" >}}
+- {{< contentlink path="projects/recipe-search/part-2" flavour="react,javascript" >}}
```

Comment:
It should be just 'React' since React itself is built with JavaScript. No need to mention both 'React' and 'JavaScript' separately.